libpod/container_inspect.go

```diff
@@ -114,18 +115,21 @@ func (c *Container) getContainerInspectData(size bool, driverData *inspect.Data)
 	// Get information on the container's network namespace (if present)
 	if runtimeInfo.NetNS != nil {
 		// Go through our IP addresses
-		ctrIPs := []string{}
-
 		for _, ctrIP := range c.state.IPs {
+			ipWithMask := ctrIP.Address.String()
+			splitIP := strings.Split(ipWithMask, "/")
+			mask, _ := strconv.Atoi(splitIP[1])
 			if ctrIP.Version == "4" {
-				ipWithMask := ctrIP.Address.String()
-				splitIP := strings.Split(ipWithMask, "/")
-				ctrIPs = append(ctrIPs, splitIP[0])
+				data.NetworkSettings.IPAddress = splitIP[0]
+				data.NetworkSettings.IPPrefixLen = mask
+				data.NetworkSettings.Gateway = ctrIP.Gateway.String()
+			} else {
```

Comment:
Probably over defensive, but would it make sense to do "else if ctrIP.Version == "6" {" here and then have the else display some kind of warning message?  A thought for consideration.